params/config.go

```diff
@@ -378,6 +381,19 @@ func (c *CliqueConfig) String() string {
 	return "clique"
 }
 
+// OptimismConfig is the optimism config.
+type OptimismConfig struct {
+	Enabled        bool           `json:"enabled"`
+	L1Block        common.Address `json:"l1_block"`
+	GasPriceOracle common.Address `json:"gas_price_oracle"`
+	FeeRecipient   common.Address `json:"fee_recipient"`
+}
+
+// String implements the stringer interface, returning the optimism fee config details.
+func (o *OptimismConfig) String() string {
+	return "optimismfee"
```

Comment:
There's a slight discrepancy between the go & json config names. I lean to just calling it `optimism` config instead of `optimismfee` in the config.

Reply:
Updated.